custom-connectors/PixelnebulaFlare/apiProperties.json

```diff
@@ -0,0 +1,36 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "B32o2Kr2AeU1w6BzY9DY",

```

Comment:
Client ID should be removed and replaced with a dummy value, or `null` for the purposes of open sourcing